prow/config.yaml

```diff
@@ -62,18 +62,17 @@ presubmits:
     - master
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/bazelbuild:0.11
+      - image: gcr.io/k8s-testimages/bazelbuild:v20170720-7762904d
         args:
-        - "--pull=$(PULL_REFS)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--job=$(JOB_NAME)"
```

Comment:
hummm I wonder if this can be default in bootstrap.py, or here it need to be `--job=pull-kubernetes-bazel`? Feels like if this is the direction to go we'll add more templates to this prow config.